software/spmd/common/spike.ld

```diff
@@ -33,7 +33,7 @@ SECTIONS
 
   .text : 
   {
-     *crt.o(.text)
+     *(.crtbegin)
```

Comment:
maybe add a comment why this change was necessary?

Reply:
done!